tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -1485,6 +1485,40 @@ TEST(http_parser, accept)
 	{
 		EXPECT_TRUE(test_bit(TFW_HTTP_B_ACCEPT_HTML, req->flags));
 	}
+
+	FOR_REQ("GET / HTTP/1.1\r\n"
+		"Accept:  invalid/invalid;  q=0.5;    key=val, */* \r\n"
+		"\r\n")
+	{
+		EXPECT_TRUE(test_bit(TFW_HTTP_B_ACCEPT_HTML, req->flags));
```

Comment:
The bit shouldn't be set for unknown values. The `TFW_HTTP_B_ACCEPT_HTML` means, that client can process HTMLs, JS code, and we can send JS code to the client and it will evaluate the code and pass our JS challenge.

Reply:
The comment is invalid, didn't noticed the `*/*` type at the end.